README.md

```diff
@@ -8,3 +8,15 @@ To build on the [Using Contentful](https://github.com/gatsbyjs/gatsby/tree/maste
 Please take a look at the [issue queue](https://github.com/mccrodp/gatsby-contentful-i18n) to help out and feel free to submit PRs :)
 
 See [DEMO](https://gatsby-starter-contentful-i18n.netlify.com/).
+
+## Quick Start
+First, create a new Gatsby site from the starter:
+
+```$ gatsby new my-fun-site https://github.com/cwlsn/gatsby-simple-contentful-starter```
+
+This starter will try to talk to a Contentful Model named homePage.
```

Comment:
Not sure what the purpose of this line is and I can't remember there being a model name "homePage" anywhere.

Reply:
Edits have been made. Thanks.